Backend/dev/migrations/rider-app/1116-add-dropLocOutsideThreshold-field-to-ride.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_app.ride ADD COLUMN drop_loc_outside_of_threshold boolean NOT NULL DEFAULT FALSE;
```

Comment:
This value can be populated instead of defaulting to false. Instead, I think its better to make it nullable and have null entries for previous rides, and why to store False while the ride is ongoing?

Reply:
if not null is involved then why it is of maybe type?